package.json

```diff
@@ -57,7 +57,8 @@
     "build:codesandbox": "yarn release:build",
     "release:changelog": "node scripts/releaseChangelog",
     "release:version": "lerna version --exact --no-changelog --no-push --no-git-tag-version",
-    "release:build": "cd packages/x-license-pro && yarn build && cd ../grid/x-data-grid-pro && yarn build && cd ../x-data-grid && yarn build && cd ../x-data-grid-generator && yarn build",
+    "release:build": "yarn release:build:next && cd packages/grid/x-data-grid-pro && yarn build && cd ../x-data-grid && yarn build && cd ../x-data-grid-generator && yarn build",
+    "release:build:next": "lerna run --parallel --scope \"@mui/x-license-pro\" build",
```

Comment:
I'm following https://github.com/mui-org/material-ui/pull/30201, https://github.com/mui-org/material-ui/pull/30200 because we will probably have the same issue.
But for now it's not a problem since I build the only package with the core bundling strategy before the others.